### DISCUSSION TITLE
Ray does not work on core php project

### DISCUSSION THREAD

**Discussion by webstar1027** [spatie/ray#413] [Q&A]
I tried to use ray debug app on my core php project.
I make ray.php on root directory of project and set ray.php as guide.
https://spatie.be/docs/ray/v1/configuration/framework-agnostic-php
And next I insert ray function on necessary part of my project, that page is broken, of course, can not debug from my ray app.
Can not make sure what is issue.

**Comment by freekmurze** [comment]
Our tests prove that Ray is working as intended. Please create a minimal PHP script for us to look at where you are having troubles getting Ray to work.
